src/com/company/administrative/User.java

```diff
@@ -0,0 +1,43 @@
+package com.company.administrative;
+
+public class User {
+    protected String username;
```

Comment:
astea sunt ok sa fie private, doar fiindca Doctor si Patient extind clasa asta, nu inseamna ca trebuie sa fie protected, si in niciintr-un caz sa nu fie accesate de catre servicii in mod direct, doar prin getteri si setteri